CHANGELOG.md

```diff
@@ -17,6 +17,31 @@ Increment the:
 
 * [RESOURCE SDK] Fix schema URL precedence bug in `Resource::Merge`.
   [#2036](https://github.com/open-telemetry/opentelemetry-cpp/pull/2036)
+* [EXPORTER] GRPC endpoint sheme should take precedence over OTEL_EXPORTER_OTLP_TRACES_INSECURE
+  [#2060](https://github.com/open-telemetry/opentelemetry-cpp/pull/2060)
+
+Important changes:
+
+* [EXPORTER] GRPC endpoint sheme should take precedence over OTEL_EXPORTER_OTLP_TRACES_INSECURE
```

Comment:
nit - scheme

Reply:
Probably is also a good idea include in the OTLP exporter doc  about `use_ssl_credentials` in https://github.com/open-telemetry/opentelemetry-cpp/blob/main/exporters/otlp/README.md with the same info about sheme precedence   described in the changelog with env vars.

Reply:
> nit - scheme

Thanks, fixed.